### PR TITLE
Add initial draft of gocumber introduction

### PR DESCRIPTION
**Pull request by ppeble**

### CODE REVIEW

**Comment by jazminschroeder** [comment]
@ptrimble 
Awesome!! I love the idea of a techblog so thanks for making it happen :)

Suggestion: A dedicated tech@sittercity.com(or something like that) as the point of contact for comments/suggestions so we all get them.

**Comment by ppeble** [comment]
We can send it to dev@sittercity.com. Where does that go currently?

**Comment by zombor** [comment]
All of us.

**Comment by ppeble** [comment]
'All of us' who? Every single person in tech? Or only developers?

File: _posts/2015-02-20-gocumber.markdown [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+func TestYourStuff(t *testing.T) {
+  steps := make(gocumber.Definitions)
+
+  steps.Given(`I have an existing user with name "(.*)"`, func(matches []string, _ gocumber.StepNode) {
+    // your code to create a user, you can use 'matches[1]' to pull out the
+    // name, like so: user_name := matches[1]
+  })
+
+  steps.When(`I deactivate the user named "(.*)"`, func(matches []string, _gocumber.StepNode) {
```
Comment (cbandy) [inline _posts/2015-02-20-gocumber.markdown]: missing space between `_gocumber`.

File: _posts/2015-02-20-gocumber.markdown [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    // your code to deactivate the specific user found in 'matches[1]'
+  })
+
+  steps.Then(`the user "(.*)" should be deactivated`, func(matches[]string, _ gocumber.StepNode) {
```
Comment (cbandy) [inline _posts/2015-02-20-gocumber.markdown]: missing space between `matches[]`.

**Comment by jazminschroeder** [comment]
hmm maybe we can create a new address tech@sittercity.com which is exclusively used for external contact and whoever is interested can subscribe to it. just throwing ideas..

File: _posts/2015-02-20-gocumber.markdown [hunk 3]
```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Gocumber"
+date:   2015-02-20 09:00:00
+categories: golang
+comments: true
+author: "Phil Trimble"
+---
+
+This month Sittercity Tech has released a new project on github: [gocumber!](https://github.com/sittercity/gocumber "gocumber")
```
Comment (cbandy) [inline _posts/2015-02-20-gocumber.markdown]: "GitHub", capitalize

**Comment by zombor** [comment]
`dev@sittercity.com` is all of tech, basically. It's probably too broad.

File: _posts/2015-02-20-gocumber.markdown (hunk 1 above)
Comment (ppeble) [inline _posts/2015-02-20-gocumber.markdown]: Addressed in https://github.com/sittercity/techblog/commit/d178bc4c14000652f3970f5f98673ce0cc59b49a

File: _posts/2015-02-20-gocumber.markdown (hunk 2 above)
Comment (ppeble) [inline _posts/2015-02-20-gocumber.markdown]: Addressed in https://github.com/sittercity/techblog/commit/d178bc4c14000652f3970f5f98673ce0cc59b49a

File: _posts/2015-02-20-gocumber.markdown (hunk 3 above)
Comment (ppeble) [inline _posts/2015-02-20-gocumber.markdown]: Addressed in https://github.com/sittercity/techblog/commit/d178bc4c14000652f3970f5f98673ce0cc59b49a

File: _posts/2015-02-20-gocumber.markdown [hunk 4]
```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Gocumber"
+date:   2015-02-20 09:00:00
+categories: golang
+comments: true
+author: "Phil Trimble"
+---
+
+This month Sittercity Tech has released a new project on Github: [gocumber!](https://github.com/sittercity/gocumber "gocumber")
```
Comment (cbandy) [inline _posts/2015-02-20-gocumber.markdown]: "GitHub", capitalize Hub.

**Comment by ppeble** [comment]
After talking with Sam we are going to remove comments and the email entirely. If people want to comment on our stuff then twitter can be their avenue.

**Comment by ppeble** [comment]
@cbandy @zombor @jazminschroeder Everyone happy with this?

**Comment by jazminschroeder** [comment]
:dog2:

**Pull request merged**